aicsimageio/readers/scn_reader.py

```diff
@@ -0,0 +1,245 @@
+                    return bool(rdr.scn_metadata)
+
+        except (TiffFileError, TypeError):
+            return False
+
+    def __init__(self, image: types.PathLike, scene: Optional[Union[int, str]] = None):
+        self._fs, self._path = io_utils.pathlike_to_fs(image, enforce_exists=True)
+        # Catch non-local file system
+        if not isinstance(self._fs, LocalFileSystem):
+            raise ValueError(
+                f"Cannot read SCN from non-local file system. "
+                f"Received URI: {self._path}, which points to {type(self._fs)}."
+            )
+
+        if not self._is_supported_image(self._fs, self._path):
+            raise exceptions.UnsupportedFileFormatError(
+                self.__class__.__name__, self._path
+            )
+
+        with TiffFile(self._path) as rdr:
+            self._scn_metadata = xml2dict(rdr.scn_metadata)
+            self._image_metadata = self._scn_metadata["scn"]["collection"]["image"]
+
+        if scene is None:
+            scene = 0
+
+        self.set_scene(scene)
+
+    @property
+    def scenes(self) -> Tuple[str, ...]:
+        scenes: Tuple[str, ...] = tuple()
+        for idx, sub_image_meta in enumerate(self._image_metadata):
+            series_meta = sub_image_meta.get("pixels").get("dimension")
+            res_indices = [d["r"] for d in series_meta]
+            n_res = np.max(res_indices) + 1
+            scenes = scenes + tuple(
+                f"{sub_image_meta.get('name')} S{idx}-(R{sr})" for sr in range(n_res)
+            )
+
+        return scenes
+
+    def _get_series_level_meta(self, series_idx: int, level_idx: int) -> Dict:
+        """tifffile organizes data series -> level (sub-resolution)
+        The metadata has similar organization but with series -> level -> channel
+        This makes sure the correct X and Y pixel lengths are pulled
+        """
+
+        series_level_meta = (
+            self._image_metadata[series_idx].get("pixels").get("dimension")
+        )
+
+        return [md for md in series_level_meta if md.get("r") == level_idx][0]
+
+    def _get_xy_pixel_sizes(
+        self, rdr: TiffFile, series_idx: int, level_idx: int
+    ) -> Dict[str, float]:
+        """Resolution data is stored in the TIFF tags in
+        Pixels per cm, this is converted to microns per pixel
+        """
+
+        current_page = rdr.series[series_idx].levels[level_idx].pages[0]
+
+        x_res = current_page.tags["XResolution"].value
+        y_res = current_page.tags["XResolution"].value
+
+        res_unit = current_page.tags["ResolutionUnit"].value
+
+        # convert units to micron
+        if res_unit.value == 1:
+            res_to_um = 1
+        if res_unit.value == 2:
+            res_to_um = 25400
+        elif res_unit.value == 3:
+            res_to_um = 10000
+
+        x_res_um = (1 / (x_res[0] / x_res[1])) * res_to_um
+        y_res_um = (1 / (y_res[0] / y_res[1])) * res_to_um
+
+        return {"X": x_res_um, "Y": y_res_um}
```

Comment:
You could return a `PhysicalPixelSizes` object here instead of Dict. More strongly typed.

Reply:
I did this throughout.